src/odemis/gui/cont/stream.py

```diff
@@ -129,8 +129,11 @@ def __init__(self, stream_bar, stream, tab_data_model, show_panel=True, view=Non
         if hasattr(stream, "power"):
             va = self.stream.power
             name = "power"
-            if self.stream.emitter:
-                hw_settings = self.tab_data_model.main.hw_settings_config
+            hw_settings = self.tab_data_model.main.hw_settings_config
+            # first check for a light component
```

Comment:
This comment maybe will add more value if in short, it can say why first light is checked?